### PR TITLE
fix: update chain integration developer guide

### PR DESCRIPTION
**Pull request by ffe9f8**
- Re-name from "overview" to "developer guide"
- Re-structure integration architecture and integration process sections for clarity and readibility
- Add current contracts versions (as of [May 14 Amplifier release notes](https://www.notion.so/bright-ambert-2bd/d60d543275c94f699102c85233257116?v=a00b1a41bceb46909b145b36290c7440&p=4b0b3efcbb15455ab71c7e46fb6c4593&pm=s))

Preview: https://axelar-docs-git-marty-amplifier-architect-974132-axelar-network.vercel.app/dev/amplifier/chain-integration/introduction

### CODE REVIEW

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 1]
```diff
@@ -1,63 +1,101 @@
-# Introduction to Chain Integration
+# Chain Integration on the Axelar Amplifier
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:1]:
```suggestion
# Integrating Your Chain Integration with Axelar via Amplifier
```

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 2]
```diff
@@ -1,63 +1,101 @@
...
-    - Ensure your relayer is robust, fault-tolerant, and can handle the load of your chain.
-1. Propose to Mainnet
-    - [Make on-chain proposals](/learn/cosmwasm-governance) on Mainnet to create your CosmWasm smart contracts.
-        1. Proposal to store your CosmWasm code.
-        1. Proposal to instantiate your contracts.
-    - Make an on-chain proposal to add your chain to the Amplifier router.
\ No newline at end of file
+## Integration architecture
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you'll need to build or instantiate a [message relayer](/dev/amplifier/chain-integration/introduction#message-relayer) service and the following four smart contracts:
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:11]: order of this list doesn't make sense when the order below is different. Maybe delete this paragraph entirely or shorten it?

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 3]
```diff
@@ -1,63 +1,101 @@
...
+
+### Message relayer
+
+- This is a service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
+- For a detailed implementation, see the [Message Relayer example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier) on GitHub.
+
+### Source chain gateway contract
+
+- This is the gateway contract for your source chain, built using the same tech stack as that chain. It serves as the primary API for developers sending messages from your chain.
+- The contract must implement the [Cross-Chain Gateway Protocol](https://github.com/axelarnetwork/cgp-spec).
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:26]: I believe @milapsheth said this repo might not be up to date anymore, and we should be pointing folks to the Solidity gateway v2.

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 4]
```diff
@@ -1,63 +1,101 @@
...
+### Axelar gateway contract
+
+- This is a smart contract on the Axelar network that checks the validity of a transaction with the verifier contract and forwards it to the Amplifier router.
+- Most integrators will begin by using or customizing the Axelar [Gateway contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/gateway) (current version: 0.2.0).
+- [Instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/gateway/src/msg.rs) can be found on GitHub.
+
+### Voting verifier contract
+
+- This is a smart contract on the Axelar network that verifies the validity of transactions on your chain.
+- Most integrators will begin by using or customizing the Axelar [Voting Verifier contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/voting-verifier) (current version: 0.3.0), which can be customized to your needs for cases such as verification via ZK Proof.
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:38]: I don't think listing versions in this doc helps. I think we should specify the versions when we are actually doing the work, not explaining it conceptually.

File: src/pages/dev/amplifier/chain-integration/introduction.mdx [hunk 5]
```diff
@@ -1,63 +1,101 @@
...
+- [Instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/voting-verifier/src/msg.rs) can be found on GitHub.
+- If you deploy a Voting Verifier contract, verifiers will need to support your chain and vote on the truth of source chain transactions.
+
+### Multisig prover contract
+
+- This is a smart contract on the Axelar network that knows how to prove the validity of a transaction on your chain.
+- Most integrators will begin by using or customizing the Axelar [Multisig Prover contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/multisig-prover) (current version: 0.3.0).
+- [Instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/multisig-prover/src/msg.rs) can be found on GitHub.
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/introduction.mdx:46]: I think the purpose of this page is a conceptual overview, not an implementation guide, so I think the specific parameters gets in the way of understanding.

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: src/pages/dev/amplifier/chain-integration/introduction.mdx (hunk 1 above)
Comment (ffe9f8) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: Changed to "developer guide"

File: src/pages/dev/amplifier/chain-integration/introduction.mdx (hunk 2 above)
Comment (ffe9f8) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: I think the order is correct?

File: src/pages/dev/amplifier/chain-integration/introduction.mdx (hunk 4 above)
Comment (ffe9f8) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: SG. Removed current versions for now. Still looking for the versioning matrix, will update this page when I find it.

File: src/pages/dev/amplifier/chain-integration/introduction.mdx (hunk 3 above)
Comment (ffe9f8) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: [This one](https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/AxelarGateway.sol)?

Comment (ffe9f8) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: Updated. Should the "EVM chains can re-use the existing EVM Gateway" part go?

File: src/pages/dev/amplifier/chain-integration/introduction.mdx (hunk 2 above)
Comment (ffe9f8) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: gateway, contracts, then relayer

Comment (StephenFluin) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: Relayer sould come last in this subsection.

File: src/pages/dev/amplifier/chain-integration/introduction.mdx (hunk 3 above)
Comment (StephenFluin) [reply src/pages/dev/amplifier/chain-integration/introduction.mdx]: That's the part that's still correct. Everyone should base off of the EVM gateway here: https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/AxelarGateway.sol

File: src/layouts/navigation.ts [hunk 6]
```diff
@@ -227,14 +227,14 @@ export const getNavigation = (section) => {
...
             },
-            {
-              title: "Integration Overview",
-              href: "/dev/amplifier/chain-integration/introduction",
-            },
             {
               title: "Integrate a Chain",
               href: "/dev/amplifier/chain-integration/integrate-a-chain",
             },
+            {
```
Comment (ffe9f8) [inline src/layouts/navigation.ts:234]: Put back above

File: src/pages/dev/amplifier/chain-integration/developer-guide.mdx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+### Message relayer
+
+- This is a service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
+- For a detailed implementation, see the [Message Relayer example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier) on GitHub.
+
+## Integration process
+
+Most integrators will do the following to onboard their chains onto the Axelar Amplifier:
+
+**1. Learn and understand Amplifier**
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/developer-guide.mdx:51]: change steps for 1.s so reordering doesn't break things / change things

File: src/pages/dev/amplifier/chain-integration/developer-guide.mdx [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+Make sure you can send and receive messages on your chain.
+
+**6. Begin audit process**
+
+Audit your on-chain contracts.
+
+**7. Propose to testnet**
+
+1. Make two on-chain proposals on the testnet to create your CosmWasm smart contracts:
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/developer-guide.mdx:77]: Fix nested numbering

File: src/pages/dev/amplifier/chain-integration/developer-guide.mdx [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+1. Make an on-chain proposal to add your chain to the Amplifier router. A router uses the governance method [`RegisterChain()`](https://github.com/axelarnetwork/axelar-amplifier/blob/main/packages/router-api/src/msg.rs#L13) to find and register new chain integrations. The chain name is then used to fund rewards and connect verifiers.
+
+## Manage the rewards pool
+
+### Check status of a rewards pool
+
+Use the following command to check the status of a rewards pool:
+
+```bash
+axelard tx wasm execute $CONTRACT_ADDRESS '' --from validator --gas auto --gas-adjustment 2
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/developer-guide.mdx:107]: Command still doesn't make sense? Doesn't do anything?

File: src/pages/dev/amplifier/chain-integration/developer-guide.mdx [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+axelard q wasm contract-state smart [rewards contract address] '{"rewards_pool":{"pool_id":{"chain_name":[chain-name],"contract":[address of voting verifier or multisig]}}}' --node $RPC
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/developer-guide.mdx:108]: --node http://devnet-amplifier.axelar.dev:26657

axelar1vaj9sfzc3z0gpel90wu4ljutncutv0wuhvvwfsh30rqxq422z89qnd989l

File: src/pages/dev/amplifier/chain-integration/developer-guide.mdx [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+```
+
+When querying for voting rewards, `contract` is the address of the voting verifier.
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration/developer-guide.mdx:111]: or prover

**Review by StephenFluin — CHANGES_REQUESTED** [review]

**Comment by ffe9f8** [comment]
New docs coming out that will cover the same things.

**Pull request closed without merging**